src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  height: 408px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  position: relative;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $text-secondary-color;
+    line-height: 14px;
+    text-align: left;
+    position: relative;
+  }
+
+  &__reviews {
+    color: $text-main-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
```

Comment:
Ensure that the variable `$text-secondary-color` is defined in your SCSS files. If not, this will cause a compilation error.